<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assertViewer</class>
 <widget class="QWidget" name="assertViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2633</width>
    <height>1322</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1761</width>
    <height>984</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>assertViewer - ${title}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="PyDMWaveformPlot" name="PyDMWaveformPlot_2">
       <property name="toolTip">
        <string/>
       </property>
       <property name="showXGrid">
        <bool>true</bool>
       </property>
       <property name="showYGrid">
        <bool>true</bool>
       </property>
       <property name="yAxes">
        <stringlist>
         <string>{&quot;name&quot;: &quot;Counts&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: &quot;ADC Counts&quot;, &quot;minRange&quot;: -10.181348642989422, &quot;maxRange&quot;: 10.181348642989422, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
        </stringlist>
       </property>
       <property name="title" stdset="0">
        <string>Y Coordinates - Plane 2</string>
       </property>
       <property name="xLabels">
        <stringlist>
         <string>Channel Index</string>
        </stringlist>
       </property>
       <property name="showLegend">
        <bool>true</bool>
       </property>
       <property name="curves">
        <stringlist/>
       </property>
       <property name="autoRangeX">
        <bool>true</bool>
       </property>
       <property name="maxXRange">
        <double>1.000000000000000</double>
       </property>
       <property name="autoRangeY">
        <bool>true</bool>
       </property>
       <property name="minYRange">
        <double>0.000000000000000</double>
       </property>
       <property name="maxYRange">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="PyDMWaveformPlot" name="PyDMWaveformPlot_7">
       <property name="toolTip">
        <string/>
       </property>
       <property name="showXGrid">
        <bool>true</bool>
       </property>
       <property name="showYGrid">
        <bool>true</bool>
       </property>
       <property name="yAxes">
        <stringlist>
         <string>{&quot;name&quot;: &quot;Counts&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: &quot;ADC Counts&quot;, &quot;minRange&quot;: -10.181348642989422, &quot;maxRange&quot;: 10.181348642989422, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
        </stringlist>
       </property>
       <property name="title" stdset="0">
        <string>X Coordinates - Plane 7</string>
       </property>
       <property name="xLabels">
        <stringlist>
         <string>Channel Index</string>
        </stringlist>
       </property>
       <property name="showLegend">
        <bool>true</bool>
       </property>
       <property name="curves">
        <stringlist/>
       </property>
       <property name="autoRangeX">
        <bool>true</bool>
       </property>
       <property name="maxXRange">
        <double>1.000000000000000</double>
       </property>
       <property name="autoRangeY">
        <bool>true</bool>
       </property>
       <property name="minYRange">
        <double>0.000000000000000</double>
       </property>
       <property name="maxYRange">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PyDMWaveformPlot" name="PyDMWaveformPlot_3">
       <property name="toolTip">
        <string/>
       </property>
       <property name="showXGrid">
        <bool>true</bool>
       </property>
       <property name="showYGrid">
        <bool>true</bool>
       </property>
       <property name="yAxes">
        <stringlist>
         <string>{&quot;name&quot;: &quot;Counts&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: &quot;ADC Counts&quot;, &quot;minRange&quot;: -10.181348642989422, &quot;maxRange&quot;: 10.181348642989422, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
        </stringlist>
       </property>
       <property name="title" stdset="0">
        <string>X Coordinates - Plane 3</string>
       </property>
       <property name="xLabels">
        <stringlist>
         <string>Channel Index</string>
        </stringlist>
       </property>
       <property name="showLegend">
        <bool>true</bool>
       </property>
       <property name="curves">
        <stringlist/>
       </property>
       <property name="autoRangeX">
        <bool>true</bool>
       </property>
       <property name="maxXRange">
        <double>1.000000000000000</double>
       </property>
       <property name="autoRangeY">
        <bool>true</bool>
       </property>
       <property name="minYRange">
        <double>0.000000000000000</double>
       </property>
       <property name="maxYRange">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="PyDMWaveformPlot" name="PyDMWaveformPlot_4">
       <property name="toolTip">
        <string/>
       </property>
       <property name="showXGrid">
        <bool>true</bool>
       </property>
       <property name="showYGrid">
        <bool>true</bool>
       </property>
       <property name="yAxes">
        <stringlist>
         <string>{&quot;name&quot;: &quot;Counts&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: &quot;ADC Counts&quot;, &quot;minRange&quot;: -10.181348642989422, &quot;maxRange&quot;: 10.181348642989422, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
        </stringlist>
       </property>
       <property name="title" stdset="0">
        <string>Y Coordinates - Plane 4</string>
       </property>
       <property name="xLabels">
        <stringlist>
         <string>Channel Index</string>
        </stringlist>
       </property>
       <property name="showLegend">
        <bool>true</bool>
       </property>
       <property name="curves">
        <stringlist/>
       </property>
       <property name="autoRangeX">
        <bool>true</bool>
       </property>
       <property name="maxXRange">
        <double>1.000000000000000</double>
       </property>
       <property name="autoRangeY">
        <bool>true</bool>
       </property>
       <property name="minYRange">
        <double>0.000000000000000</double>
       </property>
       <property name="maxYRange">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="PyDMWaveformPlot" name="PyDMWaveformPlot_6">
       <property name="toolTip">
        <string/>
       </property>
       <property name="showXGrid">
        <bool>true</bool>
       </property>
       <property name="showYGrid">
        <bool>true</bool>
       </property>
       <property name="yAxes">
        <stringlist>
         <string>{&quot;name&quot;: &quot;Counts&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: &quot;ADC Counts&quot;, &quot;minRange&quot;: -10.181348642989422, &quot;maxRange&quot;: 10.181348642989422, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
        </stringlist>
       </property>
       <property name="title" stdset="0">
        <string>Y Coordinates - Plane 6</string>
       </property>
       <property name="xLabels">
        <stringlist>
         <string>Channel Index</string>
        </stringlist>
       </property>
       <property name="showLegend">
        <bool>true</bool>
       </property>
       <property name="curves">
        <stringlist/>
       </property>
       <property name="autoRangeX">
        <bool>true</bool>
       </property>
       <property name="maxXRange">
        <double>1.000000000000000</double>
       </property>
       <property name="autoRangeY">
        <bool>true</bool>
       </property>
       <property name="minYRange">
        <double>0.000000000000000</double>
       </property>
       <property name="maxYRange">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="PyDMWaveformPlot" name="PyDMWaveformPlot_1">
       <property name="toolTip">
        <string/>
       </property>
       <property name="showXGrid">
        <bool>true</bool>
       </property>
       <property name="showYGrid">
        <bool>true</bool>
       </property>
       <property name="yAxes">
        <stringlist>
         <string>{&quot;name&quot;: &quot;Counts&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: &quot;ADC Counts&quot;, &quot;minRange&quot;: -11.452632559947652, &quot;maxRange&quot;: 11.452632559947652, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
        </stringlist>
       </property>
       <property name="title" stdset="0">
        <string>X Coordinates - Plane 1</string>
       </property>
       <property name="xLabels">
        <stringlist>
         <string>Channel Index</string>
        </stringlist>
       </property>
       <property name="showLegend">
        <bool>true</bool>
       </property>
       <property name="curves">
        <stringlist/>
       </property>
       <property name="autoRangeX">
        <bool>true</bool>
       </property>
       <property name="minXRange">
        <double>0.000000000000000</double>
       </property>
       <property name="maxXRange">
        <double>1.000000000000000</double>
       </property>
       <property name="autoRangeY">
        <bool>true</bool>
       </property>
       <property name="minYRange">
        <double>0.000000000000000</double>
       </property>
       <property name="maxYRange">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="PyDMWaveformPlot" name="PyDMWaveformPlot_5">
       <property name="toolTip">
        <string/>
       </property>
       <property name="showXGrid">
        <bool>true</bool>
       </property>
       <property name="showYGrid">
        <bool>true</bool>
       </property>
       <property name="yAxes">
        <stringlist>
         <string>{&quot;name&quot;: &quot;Counts&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: &quot;ADC Counts&quot;, &quot;minRange&quot;: -10.181348642989422, &quot;maxRange&quot;: 10.181348642989422, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
        </stringlist>
       </property>
       <property name="title" stdset="0">
        <string>X Coordinates - Plane 5</string>
       </property>
       <property name="xLabels">
        <stringlist>
         <string>Channel Index</string>
        </stringlist>
       </property>
       <property name="showLegend">
        <bool>true</bool>
       </property>
       <property name="curves">
        <stringlist/>
       </property>
       <property name="autoRangeX">
        <bool>true</bool>
       </property>
       <property name="maxXRange">
        <double>1.000000000000000</double>
       </property>
       <property name="autoRangeY">
        <bool>true</bool>
       </property>
       <property name="minYRange">
        <double>0.000000000000000</double>
       </property>
       <property name="maxYRange">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="PyDMTabWidget" name="PyDMTabWidget_config">
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alarmChannels">
        <stringlist>
         <string></string>
        </stringlist>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Configuration</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="4" column="1">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Default counts are RAW data. Uncheck boxes below to REMOVE noise components.</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="PyDMCheckbox" name="PyDMCheckbox_noise">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Noise</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="PyDMCheckbox" name="PyDMCheckbox_cm">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Common Mode</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="PyDMCheckbox" name="PyDMCheckbox_pedestals">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Pedestals</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="PyDMWaveformPlot" name="PyDMWaveformPlot_8">
       <property name="toolTip">
        <string/>
       </property>
       <property name="showXGrid">
        <bool>true</bool>
       </property>
       <property name="showYGrid">
        <bool>true</bool>
       </property>
       <property name="yAxes">
        <stringlist>
         <string>{&quot;name&quot;: &quot;Counts&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: &quot;ADC Counts&quot;, &quot;minRange&quot;: -10.181348642989422, &quot;maxRange&quot;: 10.181348642989422, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
        </stringlist>
       </property>
       <property name="title" stdset="0">
        <string>Y Coordinates - Plane 8</string>
       </property>
       <property name="xLabels">
        <stringlist>
         <string>Channel Index</string>
        </stringlist>
       </property>
       <property name="showLegend">
        <bool>true</bool>
       </property>
       <property name="curves">
        <stringlist/>
       </property>
       <property name="autoRangeX">
        <bool>true</bool>
       </property>
       <property name="maxXRange">
        <double>1.000000000000000</double>
       </property>
       <property name="autoRangeY">
        <bool>true</bool>
       </property>
       <property name="minYRange">
        <double>0.000000000000000</double>
       </property>
       <property name="maxYRange">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="PyDMTabWidget" name="PyDMTabWidget_main">
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alarmChannels">
        <stringlist>
         <string></string>
        </stringlist>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Main</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMTabWidget</class>
   <extends>QTabWidget</extends>
   <header>pydm.widgets.tab_bar</header>
  </customwidget>
  <customwidget>
   <class>PyDMWaveformPlot</class>
   <extends>QGraphicsView</extends>
   <header>pydm.widgets.waveformplot</header>
  </customwidget>
  <customwidget>
   <class>PyDMCheckbox</class>
   <extends>QCheckBox</extends>
   <header>pydm.widgets.checkbox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
